proxy/backend/Dockerfile

```diff
@@ -15,7 +15,7 @@ RUN ./gradlew --no-daemon shadowJar
 FROM eclipse-temurin:11
 WORKDIR /usr/src/app
 COPY --from=build /usr/src/app/build/libs/openlineage-proxy-*.jar /usr/src/app
-COPY proxy.dev.yml proxy.dev.yml
```

Comment:
Can we keep it `proxy.dev.yml` to indicate it is an example setup to be run locally?

Reply:
Sure thing! I'll include the changes in this file but I'll leave them commented with some instructions so people know how to proceed.